schemas/com.roku/video/jsonschema/1-0-0

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema for a Roku video player event",
+  "self": {
+    "format": "jsonschema",
+    "name": "video",
+    "vendor": "com.roku",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "videoId": {
+      "type": "string",
+      "description": "ID generated when video tracking of the video node was initialized.",
+      "maxLength": 255
+    },
+    "contentId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "Played video content ID.",
+      "maxLength": 255
+    },
```

Comment:
Whats the difference between video id and content id? Curious why this is optional too?

Reply:
Video ID is generated by the tracker when tracking for the video is initialized and remains the same for all events originating from the video. Content ID may be added by developers when setting content to be played on the video player. But it is not a required property for the video player (it is just additional metadata).